tailwind.config.js

```diff
@@ -17,6 +17,41 @@ module.exports = {
       },
     },
     extend: {
+      fontFamily: {
+        sans:["Plus Jakarta Sans",]
+      },
+      colors: {
+        // main text 'CloudBurst'
+        textBlue: "#252F56",
```

Comment:
I think we should name our custom tokens with a prefix (eg. `vd-*`) to distinguish them. We should also maintain the tailwind kebab-case for consistency.

Examples:

- `vd-text-blue`
- `vd-beige-400`

What do you think?